### ISSUE TITLE
[FavoriteMedia] Fails to detect embed permissions

### ISSUE THREAD

**Issue by QbDesu** [Dastan21/BDAddons#23]
```
[Patcher] Could not fire after callback of render for FavoriteMedia TypeError: a.isOwner is not a function
    at Object.R [as computePermissions] (8490b23c268c891a0e64.js:540)
    at H (8490b23c268c891a0e64.js:410)
    at t.n.can (8490b23c268c891a0e64.js:412)
    at Object.callback (FavoriteMedia.plugin.js:1717)
```
I'm facing the same error with https://github.com/QbDesu/BetterDiscordAddons/blob/cbf901d167d98f214a27f050a6e302d81b63cf45/Plugins/Freemoji/Freemoji.plugin.js#L376-L381
and I remembered your plugin also needs to check for embed perms so I tested it really quick and it also runs into an error

**Comment by Dastan21** [comment]
Fixed in a204c3d with a custom function (see https://github.com/Dastan21/BDAddons/blob/a204c3d6012e3856e4f318509f20660f43410107/plugins/FavoriteMedia/FavoriteMedia.plugin.js#L4286-L4298).

**Comment by QbDesu** [comment]
Apparently the easier solution was just that the second and third parameter were swapped.
Now it requires `Permissions.can(<permission>, <channel>, <user>)` instead of `Permissions.can(<permission>, <user>, <channel>)`

**Comment by Dastan21** [comment]
Ho ok ty, will fix later ;)

**Comment by Dastan21** [comment]
Fixed in 8fe5927
